docs/javascripts/version-select.js

```diff
@@ -0,0 +1,54 @@
+window.addEventListener("DOMContentLoaded", function() {
+
+  // ABS_BASE_URL is the full URL
+  // e.g. https://acquire-project.github.io/acquire-docs/dev/get_started/
+  var ABS_BASE_URL = document.baseURI;
+  // This should probably not be hardcoded
+  var CURRENT_VERSION = ABS_BASE_URL.split("/")[4];
```

Comment:
Can we do a regex search for a semver string (or "dev") instead? I agree, I don't like that this is hardcoded. `(\d+\.\d+\.\d+(\-?rc\d+)?|dev)` might do the trick:
![image](https://github.com/acquire-project/acquire-docs/assets/844464/222e5619-faf6-4c1f-a53b-2f79f78c6ee0)